app/webpacker/javascripts/markdown-preview.js

```diff
@@ -0,0 +1,28 @@
+import $ from 'jquery';
+import 'jquery.autosize' ;
+
+$(document).ready(function(){
+  var Previewer = {
```

Comment:
I think if this is our code we could do with writing some tests for this.

Reply:
Yeh - I started adding in stimulus and jest but figured that can come later - this is very much the tactical solution